test/helpers/cilium.go

```diff
@@ -574,7 +574,7 @@ func (s *SSHMeta) PolicyWait(revisionNum int) *CmdRes {
 // purposes.
 func (s *SSHMeta) ReportFailed(commands ...string) {
 	wr := s.logger.Logger.Out
-	fmt.Fprint(wr, "===================== TEST FAILED =====================\n")
+	fmt.Println("===================== TEST FAILED =====================")
```

Comment:
This won't be printed in the logger, can you make a `fmt.Fprintf` to print it in the logs?

Reply:
OK, I'll keep the old one.